html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="author" title="akansha2608" href="mailto:akanshasinghal999@gmail.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-fgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
```

Comment:
Please add an explicit body tag before this script element. Otherwise, the test might run before a body is created causing it to randomly fail.

Reply:
div implies body though

Reply:
Right. Working with Akansha on this in Slack. Stay tuned, better body coming:)